test/Riok.Mapperly.Tests/Mapping/UserMethodTest.cs

```diff
@@ -190,7 +190,7 @@ public Task WithClassBaseTypeShouldWork()
             using System.Collections.Generic;
             using Riok.Mapperly.Abstractions;
 
-            [Mapper]
+            [Mapper(EnabledConversions = MappingConversionType.ExplicitCast)]
```

Comment:
Rather than enabling the explicit cast everywhere, I think it’s better to update the test, at least in cases where it’s not specifically verifying the explicit cast behavior.